app/views/statistics/infringing_urls.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="wrapper">
+	<%= render partial: 'sidebar', locals: { sidebar_links: @sidebar_links, sidebar_items: @sidebar_items, current_opt: "infringing urls" } %>
+	<div class="main-panel">
+		<%= render partial: 'stat_navbar', locals: { navbar_list: @navbar_list, navbar_link: @navbar_link } %>
+		<div class="content">
+			<div class="container-fluid">
+				<div class="row">
+					<div class="col-md-6">
+						<%=render partial: 'number_of', locals: { display_title: "Total number of URLs", display_subtitle: "Approximate number of infringing URLs reported on Lumen", display_count: @urls_count } %>
+					</div>
+					<div class="col-md-6">
+						<%=render partial: 'number_of', locals: { display_title: "Average number of URLs", display_subtitle: "This denotes the average number of URLs per notice", display_count: @average_count, cached: true } %>
```

Comment:
This `cached: true` syntax, while slick, wasn't introduced until Rails 5, and we're running 4.2.

Reply:
Updated. Thank you!